plugin.video.srfplaytv/addon.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.srfplaytv" name="SRF Play TV" version="0.9.2" provider-name="Alexander Seiler">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.module.simplecache" version="1.0.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">SRF Play television video library</summary>
+    <summary lang="de">SRF Play Videobibliothek</summary>
+    <description lang="en_GB">SRF Play TV allows you to watch the content from the video library of Swiss Radio and Television (SRF).</description>
+    <description lang="de">Mit SRF Play TV können Sie den Inhalt der SRF Play Videobibliothek des Schweizer Fernsehens anschauen.</description>
+    <disclaimer lang="en_GB">Some content is geo-blocked and only available in Switzerland.</disclaimer>
+    <disclaimer lang="de">Einige Inhalte sind nur in der Schweiz verfügbar.</disclaimer>
+    <summery lang="de"></summery>
+    <language>de en</language>
```

Comment:
This refers to the language of the content. Are the videos in english? If not keep only `de`